ar71xx: Add usb support to Archer C7 v4 and v5 by default

halms opened the pull request:
In my opinion, the default usb packages kmod-usb-core kmod-usb2 kmod-usb-ledtrig-usbport should also be added for Archer C7 v4 and v5 by default.

(I just ran into issues with a new C7 v4 and a custom image built with the imagebuilder).

ldir-EDB0 commented:
A few points - this PR now has merge conflicts which need to be resolved.
The PR title suggests it changes the Archer c7 v4 & v5, but I only see changes for v4 device.
Finally the commit doesn't have a valid 'signed off by' 

From https://openwrt.org/submitting-patches

all commits must contain Signed-off-by: My Name <my@email.address> where you write your real name and real email address, in accordance with Section 11 of the Linux Kernel patches guide.

ldir-EDB0 commented:
commit still not signed off
PR now has two commits - the original and the merge - your original commit should be rebased on top of master and force push the PR.
c2 v5 still appears unchanged

halmi commented:
I‘m aware of the missing signoff.

The changes up to now were easily done in the GitHub-Webinterface. But in order to rebase/squash/signoff I have to use a local Git environment. Will do that once I find some time at my personal machine.

Regarding C7 v5: This does not need any edits as it uses C7 v4 as basis and therefore the same package list.

ldir-EDB0 commented:
Great - I shall keep an eye out for the changes... and yes I see the v5 device references v4 now...  $(Device/archer-c7-v4)

Commit:
ar71xx: Add usb drivers in Archer C7 v4/v5 images

This commit adds the default usb packages
- kmod-usb-core
- kmod-usb2
- kmod-usb-ledtrig-usbport
for Archer C7 v4 and v5.
(The C7 v5 configuration is based on the v4, therefore the change for v4
also applies for v5.)

Signed-off-by: Daniel Halmschlager <dh@dev.halms.at>

halmi commented:
I think everything should be fine now.

ynezz commented:
@halmi Yes, much better now. Now you just need to add answer to `Why do you need this change?` to your commit message. We can see what you're changing in the files from the diff, so you don't need to repeat that yourself, but rather spent time explaining why are you doing it. In other words, you've to convince the reviewers, that this change is needed to be included.

ldir-EDB0 commented:
Pulled into my staging tree at https://git.openwrt.org/openwrt/staging/ldir.git
